hack/manage-secrets.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -euo pipefail
+
+decrypt_secrets(){
+    local github_token_path="${1}"
+
+    target_dir=$(mktemp -d)
+    git clone https://kubevirt-bot:$(cat ${github_token_path})@github.com/kubevirt/secrets ${target_dir}
```

Comment:
Not sure if that makes sense here, but maybe we should use GIT_ASKPASS here instead of catting the token. In case one accidentally calls the function with `-x`, we then might leak the token here, might we not?

Reply:
Absolutely, makes total sense. Already pushed the changes to use `GIT_ASKPASS`, this successful execution was run with them https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/logs/post-project-infra-kuberhealthy-deployment/1362077653179305984 PTAL